tests/unit/core/test_tracking.js

```diff
@@ -27,25 +31,25 @@ import {
   UNINSTALL_THEME_CATEGORY,
 } from 'core/constants';
 
+function stubConfig(overrides = {}) {
```

Comment:
I think you should use `getFakeConfig()` from `tests/unit/helpers`.

Reply:
I used the convention already established in this file. Would it be best to update all of these in one go independently?